### DISCUSSION TITLE
Offline documentation

### DISCUSSION THREAD

**Discussion by CrazyBoyLanZao** [leapmotion/UnityModules#1320] [Q&A] [answered]
Excuse me, Where can i find the v6 offline document.  Can you tell me, thank you.

**Comment by MattGrayUL** [comment, marked as answer]
Hi @CrazyBoyLanZao

We don't provide offline documentation for v6 as it is being regularly updated online and may become out of date at any point. Please find the online documentation for v6.X of the Unity Plugin [here](https://docs.ultraleap.com/unity-api/)

Is there a particular reason you would like offline documentation?

**Reply by CrazyBoyLanZao** [reply]
Because Sometimes I can not connect the website of API, so I want to get a offline document

**Reply by MattGrayUL** [reply]
I understand, unfortunately we don't have any available offline copies of the v6.X documentation at this time

I will feed this back to the team, thanks for letting us know that you would like this option :)
